src/components/AlertBanner/AlertBanner.scss

```diff
@@ -70,6 +71,6 @@
   min-width: 0;
 }
 
-.l3-alert-banner{
+.l3-alert-banner {
```

Comment:
it's not our fonts, our fonts starts with --typography, please change it and use correctly font